File: Printpress.UI/Printpress_Angular/src/app/features/reports/models/zero-orders-report.dto.ts
export interface ZeroOrderReportRowDto {
  orderId: string;
  orderName: string;
  clientName: string;
  createdAt: string;
  status: string;
  serviceCount: number;
  itemCount: number;
  totalPrice: number;
}

export interface ZeroOrdersReportDto {
  orders: ZeroOrderReportRowDto[];
  orderCount: number;
  itemCount: number;
}

File: Printpress.Application/Reports/DTO/ZeroOrdersReportDto.cs
namespace Printpress.Application;

public class ZeroOrderReportRowDto
{
    public Guid OrderId { get; set; }
    public string OrderName { get; set; }
    public string ClientName { get; set; }
    public DateTime CreatedAt { get; set; }
    public string Status { get; set; }
    public int ServiceCount { get; set; }
    public int ItemCount { get; set; }
    public decimal TotalPrice { get; set; }
}

public class ZeroOrdersReportDto
{
    public List<ZeroOrderReportRowDto> Orders { get; set; } = [];
    public int OrderCount { get; set; }
    public int ItemCount { get; set; }
}
